documentation/homework/todo-app/src/component/footer.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export default class Footer extends React.Component {
```

Comment:
This is really cool! Going beyond the homework requirements and creating nice functionality.